src/main/java/io/jenkins/plugins/restlistparam/logic/ValueResolver.java

```diff
@@ -53,25 +59,21 @@ private ValueResolver() {
 
       NodeList nodeList = (NodeList) xPath.evaluate(expression, xmlDocument, XPathConstants.NODESET);
       if (nodeList.getLength() > 0) {
-        container.setValue(xmlNodeListToList(nodeList));
-      }
-      else {
+        container.setValue(xmlNodeListToList(nodeList, displayExpression));
+      } else {
```

Comment:
looks like your IDE is not too pleased with my mix of C# and java syntax :sweat_smile: I really should look into a linter and  maybe export my IntelliJ settings to the repo :thinking:

Reply:
oh yeah, I always mess this up. I tried to adapt, but sometimes I miss things like this

Reply:
don't worry about it, this is not blocking, just me noticing that I should do something about this. :+1: